### PR TITLE
created table

### PR DESCRIPTION
**Pull request by lucycpu**
I did the work, please review and let me know if any fixes needed.

### CODE REVIEW

File: table.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+create table dbo.germ(germid int not null identity primary key)
```
Comment (lucycpu) [inline table.sql:1]: you need more columns

Comment (charliecpu) [inline table.sql:1]: you need more columns to implement the spec

Comment (charliecpu) [reply table.sql]: I see you did it in the next commit

File: table.sql [hunk 2]
```diff
@@ -1 +1 @@
-create table dbo.germ(germid int not null identity primary key)
+create table dbo.germ(germid int not null identity primary key, germname varchar(100))
```
Comment (charliecpu) [inline table.sql:1]: still need more columns for the rest of the spec

Comment (charliecpu) [inline table.sql:1]: also need sample data and queries

**Review by charliecpu — CHANGES_REQUESTED** [review]

File: table.sql (hunk 2 above)
Comment (lucycpu) [reply table.sql:1]: added files for that

Comment (charliecpu) [reply table.sql:1]: 👍

Comment (charliecpu) [reply table.sql:1]: 👍

File: reports.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+show me....
```
Comment (charliecpu) [inline reports.sql:1]: need more detail

File: sample data.sql [hunk 4]
```diff
@@ -0,0 +1,2 @@
+common cold, ....
```
Comment (charliecpu) [inline sample data.sql:1]: need columns, and at least 5 rows

**Review by charliecpu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**